pkg/kubelet/apis/config/validation/validation.go

```diff
@@ -31,6 +32,10 @@ import (
 	kubetypes "k8s.io/kubernetes/pkg/kubelet/types"
 )
 
+var (
+	zeroDuration = metav1.Duration{}
```

Comment:
Is this already defined elsewhere?

Reply:
A couple of places but it's local to those packages. It's the default value so I didn't want to introduce another dependency.

Reply:
k sg